ZipBuilder/Sources/ZipBuilder/ZipBuilder.swift

```diff
@@ -360,24 +360,6 @@ struct ZipBuilder {
       fatalError("Could not write README to Zip directory: \(error)")
     }
 
-    // TODO: Remove this manual copy once FirebaseCrash is removed from the Zip file.
-    // Copy over the Crash scripts, if Crash should be installed
-    if subspecsToInstall.contains(.crash) {
-      do {
-        let crashDir = paths.templateDir.appendingPathComponent(Constants.ProjectPath.crashDir)
```

Comment:
Nit: there should be a few other things here you can delete: `Constants.ProjectPath.crashDir`, and all the files in [`ZipBuilder/Template/Crash`](https://github.com/firebase/firebase-ios-sdk/tree/master/ZipBuilder/Template/Crash) directory.

Reply:
Done. PTAL